### DISCUSSION TITLE
Straight strokes without using the line command

### DISCUSSION THREAD

**Discussion by oavg06** [xournalpp/xournalpp#5048] [Q&A]
Best regards, for a long time I have used the sketchbook application, in which I could make the stroke vertical or horizontal by pressing the shift key without having to activate the line command. I was exploring the options and reviewing the forum but I have not found how to do it. I only found that the shift key helps me to make the rectangle option square

**Comment by rolandlo** [comment]
You may consider using the shape recognizer. If the line you draw is nearly straight, it will be straightened.

**Reply by oavg06** [reply]
Thanks for the reply. I already tried the option, but when the stroke is short sometimes it doesn't work, also it turns the stroke into a line of uniform thickness. The idea is to activate something similar to Autocad's orthogonality. I don't know if that is possible at the moment

**Reply by profmatxavier** [reply]
Também sinto falta deste recurso no Xournal. Seria ótimo fazer retas precionsndo a tecla Swift.

**Comment by netlimpopo** [comment]
> Best regards, for a long time I have used the sketchbook application, in which I could make the stroke vertical or horizontal by pressing the shift key without having to activate the line command. I was exploring the options and reviewing the forum but I have not found how to do it. I only found that the shift key helps me to make the rectangle option square<

You can adjust the minimum distance, at which the shape recogniser starts to interpret the pen input as a line under" settings", in order to draw short lines. In a note taking app it makes more sense to make it a straight line with constant thickness. Unfortunately it's impossible to gain a pressure sensitive line (shape) from the shape recogniser at the moment.

**Reply by oavg06** [reply]
Gracias por la respuesta. Esa parte ya a miré y funciona para que convierta el trazo en línea, pero no siempre queda horizontal.
